amazon-kinesis/1.0/modules/ROOT/pages/amazon-kinesis-connector-config-topics.adoc

```diff
@@ -0,0 +1,76 @@
+= Amazon Kinesis Data Streams Connector 1.0 Additional Configuration Information - Mule 4
+
+== Aggregating Records
+
+When you use the *Put Record* operation, you can aggregate the records that you put into the Kinesis data stream he following fields for the *Put Record* operation
+
+* *Put Record*, which puts a record into the Kinesis data stream.
++
+If the *Aggregation Enabled* field is `True (the default)`, then multiple records are combined and stored in one Kinesis Data Streams record. For more information on how to set up aggregation, see xref:amazon-kinesis-connector-config-topics.adoc[Additional Configuration Information].
+
+
+
+
+
+
+image::amazon-kinesis-access-configuration.png[]
+
+
+== Advanced security settings
+Apart from the Access Key, Secret Key and Region, there are several additional fields you can set up if you wish to.
+
+=== Role
+[%header,cols="30a,10a,60a",width=100%]
+|===
+|Field name|Default Value|Meaning
+|ARN|N/A| The Amazon Resource Name of the role to assume.
+|ExternalId|N/A| A unique identifier that might be required when you assume a role in another account. If the administrator of the account to which the role belongs provides an external ID, provide that value in the ExternalId parameter.
+|Duration|3600| The duration of the role session.
+|Duration Time Unit|Seconds| The time unit for the *Duration* field.
+|Referred Policy ARNs|N/A| The Amazon Resource Names (ARNs) of the IAM managed policies that you want to use as managed session policies. The policies must exist in the same account as the role.
+|Tags|N/A| A list of session tags that you want to pass.
+|===
+
+
+== Optional fields and meanings
+This section describes the basic optional fields, their defaults and meanings.
+
+=== General section
+[%header,cols="30a,10a,60a",width=100%]
+|===
+|Field name|Default Value|Meaning
+|Default Stream Name|N/A| Name of the stream to use by default if an operation doesn't specify any.
+|Default Application Name|N/A| Name of the application to use by default if an operation that requires it doesn't specify one.
+|===
+
+=== Proxy:
```

Comment:
is it use on producer and consumer side? otherwise we need to update this and/or move if to consumer/produce specific tabs @adam-lipka

Reply:
Yes, confirmed above.